test/views/amendment/AmendRegistrationViewSpec.scala

```diff
@@ -100,6 +100,17 @@ class AmendRegistrationViewSpec extends UnitViewSpec with Matchers {
             }))
           }
 
+          "display the change-group-lead-link correctly" in {
+            val maybeLink = Option(view.select("#change-group-lead-link").get(0))
+            organisationType match {
+              case Organisation =>
+                maybeLink mustBe defined
+                maybeLink.get.text() mustBe messages("amend.partnership.changeGroupLead.link")
```

Comment:
can this be `amend.group.changeGroupLead.link` please

Reply:
```suggestion
                maybeLink.get.text() mustBe messages("amend.group.changeGroupLead.link")
```